src/Test/SymfonyTest.php

```diff
@@ -108,14 +108,21 @@ protected function getProxy()
      *
      * @return Symfony
      */
-    protected function getProxyClient()
+    protected function getProxyClient(array $options = array())
     {
+        $options = array_merge(
+            [
+                'purge_method' => 'NOTIFY'
```

Comment:
Actually I introduced the passing of `$options` to this method in order to be able to pass in the Invalidator.